registry/coder/modules/agentapi/main.tf

```diff
@@ -0,0 +1,277 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    coder = {
+      source  = "coder/coder"
+      version = ">= 2.7"
+    }
+  }
+}
+
+variable "agent_id" {
+  type        = string
+  description = "The ID of a Coder agent."
+}
+
+data "coder_workspace" "me" {}
+
+data "coder_workspace_owner" "me" {}
+
+variable "web_app_order" {
+  type        = number
+  description = "The order determines the position of app in the UI presentation. The lowest order is shown first and apps with equal order are sorted by name (ascending order)."
+  default     = null
+}
+
+variable "web_app_group" {
+  type        = string
+  description = "The name of a group that this app belongs to."
+  default     = null
+}
+
+variable "web_app_icon" {
+  type        = string
+  description = "The icon to use for the app."
+}
+
+variable "web_app_display_name" {
+  type        = string
+  description = "The display name of the web app."
+}
+
+variable "web_app_slug" {
+  type        = string
+  description = "The slug of the web app."
+}
+
+variable "folder" {
+  type        = string
+  description = "The folder to run AgentAPI in."
+  default     = "/home/coder"
+}
+
+variable "cli_app" {
+  type        = bool
+  description = "Whether to create the CLI workspace app."
+  default     = false
+}
+
+variable "cli_app_order" {
+  type        = number
+  description = "The order of the CLI workspace app."
+  default     = null
+}
+
+variable "cli_app_group" {
+  type        = string
+  description = "The group of the CLI workspace app."
+  default     = null
+}
+
+variable "cli_app_icon" {
+  type        = string
+  description = "The icon to use for the app."
+  default     = "/icon/claude.svg"
+}
+
+variable "cli_app_display_name" {
+  type        = string
+  description = "The display name of the CLI workspace app."
+}
+
+variable "cli_app_slug" {
+  type        = string
+  description = "The slug of the CLI workspace app."
+}
+
+variable "pre_install_script" {
+  type        = string
+  description = "Custom script to run before installing the agent used by AgentAPI."
+  default     = null
+}
+
+variable "install_script" {
+  type        = string
+  description = "Script to install the agent used by AgentAPI."
+  default     = ""
+}
+
+variable "post_install_script" {
+  type        = string
+  description = "Custom script to run after installing the agent used by AgentAPI."
+  default     = null
+}
+
+variable "start_script" {
+  type        = string
+  description = "Script that starts AgentAPI."
+}
+
+variable "install_agentapi" {
+  type        = bool
+  description = "Whether to install AgentAPI."
+  default     = true
+}
+
+variable "agentapi_version" {
+  type        = string
+  description = "The version of AgentAPI to install."
+  default     = "v0.2.3"
```

Comment:
Is there a way to make this `latest` and fetch the latest release version?
We have a similar logic in `vault-github` module.

https://github.com/coder/registry/blob/c0f2d945c5808c1962b15b5c2e2a5269ea0f4339/registry/coder/modules/vault-github/main.tf#L35-L43

Reply:
Yes - I'd like to avoid that. I don't want `latest` to be the default value for the reasons I stated above.